frameworks/g-cloud-9/questions/services/supportedBrowsers.yml

```diff
@@ -0,0 +1,37 @@
+name: Supported browsers
+question: 'What browsers does your service work with?'
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+      - cloud-support
+hint: 'Choose all that apply.'
+type: checkboxes
+options:
+    -
+        label: 'Internet Explorer 6'
```

Comment:
Now! It's been removed. Spoke to Paul Hayes and gov admin tools are 'functional' on IE 7 – so we’re removing IE6 🎉